src/Hl7.Fhir.Specification.Tests/SerializationInfoTestsHelpers.cs

```diff
@@ -175,5 +175,21 @@ int fail(string message)
             }
 
         }
+
+        public static void TestXmlRepresetation(IStructureDefinitionSummaryProvider provider)
+        {
+            var testElements = new XmlRepresentation[] { XmlRepresentation.XmlAttr, XmlRepresentation.TypeAttr, XmlRepresentation.XHtml }; // Elements defined by TestXmlRepresentation (Logical Model)
+            var summary = provider.Provide("http://hl7.org/fhir/StructureDefinition/TestXmlRepresentation");
+            var summaryElements = summary.GetElements();
+
+            foreach(var element in testElements)
+            {
+                var representation = summaryElements.Where(e => e.ElementName == element.ToString()).FirstOrDefault().Representation;
+                if (!representation.ToString().Equals(element.ToString()))
```

Comment:
Why not like this:
`if (!representation.Equals(element))`

Reply:
Of course... Changed as requested